vm-setup/roles/v1aX_integration_test/tasks/move.yml

```diff
@@ -29,6 +30,9 @@
           
       - name: Fetch container logs before pivoting (kind cluster)
         shell: "sudo {{ CONTAINER_RUNTIME }} logs {{ item }} > /tmp/{{ CONTAINER_RUNTIME }}/{{ item }}/stdout.log 2> /tmp/{{ CONTAINER_RUNTIME }}/{{ item }}/stderr.log"
+        # FIXME(dtantsur): remove ignore_errors after the transition period
+        # from ironic-api+conductor to just ironic
```

Comment:
Am I getting it right here, that the end goal is to have only one container and get rid of ironic-api+conductor?